.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,95 @@
+name: 🐞 Bug report
+description: Create a report to help us improve.
+labels: ["bug", "triage"]
+body:
+  - type: input
+    id: jerboaversion
+    attributes:
+      label: Jerboa Version
+      description: Type the version from the settings here.
+      placeholder: ex. 0.0.39
+    validations:
+      required: true
+  - type: input
+    id: androidversion
+    attributes:
+      label: Android Version + Phone
+      description: Type the android version you are using and which phone here.
+      placeholder: ex. Android 13, Google Pixel 7 Pro
+    validations:
+      required: true
+  - type: textarea
+    id: describebug
+    attributes:
+      label: Describe The Bug
+      description: A clear and concise description of what the bug is.
+      placeholder: ex. This should happen, but that happens instead.
+    validations:
+      required: true
+  - type: textarea
+    id: reproducebug
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
+  - type: textarea
+    id: incaseofcrash
+    attributes:
+      label: In the case of a crash
+      description: Please include the logs.
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        <details><summary>Click here if you don't know how to get the logs</summary>
+
+
```

Comment:
Instead of this, there's now a crash logs helper. Tell them to go to `settings -> about -> Crash Logs`

Reply:
Keep that one, cuz the crash log helper wont help if the crash is on startup. And we have had a handfull of those.

Reply:
should I just add some text in this summary dropdown for the crash log helper? or replace it altogether?

Reply:
Tell them to use the previous method if the crash is on startup else use the crash logs

Reply:
see latest comment in PR, added two sections
![image](https://github.com/dessalines/jerboa/assets/12457072/17e5a683-e6bc-4ba0-ac9a-cbe7d3f56f1b)
**FOR CRASHES ON STARTUP** has the old content
**FOR GENERAL CRASHES** has this:
![image](https://github.com/dessalines/jerboa/assets/12457072/bda79777-42ca-417a-a8a9-4f8b1a2e31e5)